### PR TITLE
Move New Search Experience out of beta

### PR DESCRIPTION
**Pull request by justjanne**
Type: Enhancement
Related: https://github.com/vector-im/element-web/issues/22553

---
Here's what your changelog entry will look like:

## ✨ Features
 * Move New Search Experience out of beta ([\#8859](https://github.com/matrix-org/matrix-react-sdk/pull/8859)). Contributed by @justjanne.

### CODE REVIEW

**Comment by luixxiul** [comment]
Let's not forget to remove `.mx_BetaCard_betaPill` inside `_SpotlightDialog.scss` as well :-)

**Comment by luixxiul** [comment]
Let's not forget to remove `.mx_BetaCard_betaPill` inside `_SpotlightDialog.scss` as well :-)

**Comment by weeman1337** [comment]
Does it make sense to keep a labs setting? I cannot uncheck it anyway

image

**Comment by justjanne** [comment]
It shouldn’t be in there anymore, thanks for finding that!

File: src/components/structures/RoomSearch.tsx [hunk 1]
```diff
@@ -43,220 +34,49 @@ interface IProps {
...
-    };
-
-    public focus = (): void => {
-        this.elementRef.current?.focus();
-    };
-
     public render(): React.ReactNode {
         const classes = classNames({
             'mx_RoomSearch': true,
-            'mx_RoomSearch_hasQuery': this.state.query,
```
Comment (weeman1337) [inline src/components/structures/RoomSearch.tsx:178]: Some of these classes doesn't seem to be used anymore but they are still around in the Sass files. Can you chat that please and eventually remove them?

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Tested on the Netlify Deployment. Seems to be working. :+1:   
Can you check my comment in the code about the unused Styles?

**Review by weeman1337 — APPROVED** [review]
:+1:

**Comment by turt2live** [comment]
(there's also docs in element-web's labs.md which might need deleting)

**Review by kittykat — APPROVED** [review]

**Comment by kittykat** [comment]
> Does it make sense to keep a labs setting? I cannot uncheck it anyway
> 
> image

I think it would be better removed

**Review by niquewoodhouse — APPROVED** [review]
🚀

**Pull request merged**